features/ForceExpiration.php

```diff
@@ -86,23 +82,67 @@ private function _addPasswordExpirationCheck() : void {
 	/**
 	 * Retrieve the password expiration date
 	 * @param User $user
-	 * @return DateTime
+	 * @return DateTime Most recent time of password change
```

Comment:
The most recent known password change is modified by the expiration window (`$mostRecent->modify("{$this->_passwordLifetime} day");`), so this is returning the expiration date, not the most recent time of password change.